setup.py

```diff
@@ -75,13 +75,11 @@
         "Intended Audience :: Developers",
         "License :: OSI Approved :: Apache Software License",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.7",
```

Comment:
Dropping support for Python 2.7 is definitely a breaking change.  ISTM that #313 (where `synthtool` adds 3.9 support to `noxfile.py`, without dropping 3.7) suggests that we aren't yet ready to do that.  @crwilcox WDYT?

Reply:
I see that @crwilcox actually [commented already](https://github.com/googleapis/python-storage/pull/295#issuecomment-725695994) that we shouldn't drop 2.7 support.